src/Nethermind/Nethermind.Blockchain/TransactionPools/Storages/NoTransactionStorage.cs

```diff
@@ -0,0 +1,21 @@
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Dirichlet.Numerics;
+
+namespace Nethermind.Blockchain.TransactionPools.Storages
+{
+    public class NoTransactionStorage : ITransactionStorage
```

Comment:
name it NullTransactionSTorage and make it a singleton - this is our convention for null classes / Null object pattern